process/project_proposals.adoc

```diff
@@ -0,0 +1,118 @@
+== OpenSSF Project Proposal Process v1.3
+
+=== Introduction
+
+This governance policy sets forth the proposal process for projects to be accepted into the Open Source Security Foundation (“OpenSSF”). The process is the same for both existing projects which seek to move into the OpenSSF, and new projects to be formed within the OpenSSF.
+
+=== Project stages
+
+image::project-stages.png[Project stages]
```

Comment:
GitHub now supports mermaid diagrams, we should use one of those instead of this PNG